<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsUI</class>
 <widget class="QDialog" name="SettingsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>409</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>591</width>
    <height>343</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройка графики</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="schemeGBox">
     <property name="title">
      <string>Схема</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="schemeCBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>По умолчанию</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveAsBtn">
        <property name="text">
         <string>Сохранить как...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGraphicsView" name="view">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>280</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="elemGBox">
         <property name="title">
          <string>Элемент схемы</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QComboBox" name="elemCBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Норма</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ухудшение</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Отказ</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Выключен</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Не обслуживается</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Подложка</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Местное</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QPushButton" name="elemColorBtn">
              <property name="minimumSize">
               <size>
                <width>142</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Цвет элемента</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="elemFontColorBtn">
              <property name="minimumSize">
               <size>
                <width>142</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Цвет текста</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="headerFontBtn">
           <property name="minimumSize">
            <size>
             <width>152</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Шрифт заголовка схемы</string>
           </property>
           <property name="text">
            <string>Шрифт заголовка</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="headerColorBtn">
           <property name="minimumSize">
            <size>
             <width>152</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Цвет заголовка схемы</string>
           </property>
           <property name="text">
            <string>Цвет заголовка</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="mainFontBtn">
           <property name="minimumSize">
            <size>
             <width>152</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Шрифт текста элементов схемы</string>
           </property>
           <property name="text">
            <string>Шрифт осн.текста</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="lineColorBtn">
           <property name="minimumSize">
            <size>
             <width>152</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Цвет линий схемы</string>
           </property>
           <property name="text">
            <string>Цвет линий</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="dlgFontBtn">
           <property name="minimumSize">
            <size>
             <width>152</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Шрифт текста в диалоговых окнах</string>
           </property>
           <property name="text">
            <string>Шрифт диалогов</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="bgColorBtn">
           <property name="minimumSize">
            <size>
             <width>152</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Цвет фона схемы</string>
           </property>
           <property name="text">
            <string>Цвет фона</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="scaleGBox">
         <property name="title">
          <string>Масштаб</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="scaleXLbl">
            <property name="text">
             <string>По X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="scaleXSBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Масштаб схемы по горизонтали</string>
            </property>
            <property name="minimum">
             <number>60</number>
            </property>
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="scaleYLbl">
            <property name="text">
             <string>По Y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="scaleYSBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Масштаб схемы по вертикали</string>
            </property>
            <property name="minimum">
             <number>60</number>
            </property>
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Применить изменения и выйти</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Применить изменения</string>
       </property>
       <property name="text">
        <string>Применить</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Отменить изменения и выйти</string>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
